content/posts/introducing-latest-aliases.md

```diff
@@ -0,0 +1,31 @@
+title: Introducing latest aliases for operating system and tools in the configuration file
+date: 2024-02-15
+category: changelog
+tags: configuration file
+authors: Santos Gallegos
+status: published
+
+While some projects benefit from the stability provided by using a specific version,
+and updating when needed, others projects, especially those under active development,
+prefer to use the latest versions available to take advantage of new features and improvements
+without having to update the configuration file every time a new version is available.
```

Comment:
If we want to keep it shorter, we can remove this paragraph.

Reply:
I think it adds a good amount of background actually. I'd keep it, but I'd suggest breaking it up a bit more to keep it easy to read.

Reply:
Could maybe use at least one period :)

Reply:
```suggestion
We generally recommend projects pin specific versions of their dependencies,
so that you have [reproducible builds](https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html) across time.
However, some projects under active development prefer to use the latest versions available to take advantage of new features and improvements,
without having to update the configuration file every time a new version is available.
```